plaso/engine/artifact_filters.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+"""Artifact filters processing."""
```

Comment:
"Artifact filters processing" => Helper to create filters based on artifact definitions.

Reply:
Done

Reply:
Merged in #1883